src/main/java/karuna/karuna_backend/content/infrastructure/ContentController.java

```diff
@@ -31,4 +36,14 @@ ContentDTO getContentByPage(@PathVariable String page) {
         return contentService.getContentByPage(page);
     }
 
+
+    @Operation(summary = "Add or update Content", description = "Add Content or update it if row with same page and key exist.")
+    @ApiResponse(
+            responseCode = "201",
+            description = "Added or updated if Content exist",
+            content = @Content(mediaType = "application/json", schema = @Schema(implementation = ContentResponseDTO.class)))
```

Comment:
Reffering to my previous comment. I'd like to modifiy this issue so it accepst List of contenst instead of only 1 becasue it is kinda non-practical. So you can easily refactor @misterrevo logic of editing content and it's ContentWrapper. Default responseCode should indeed be 201 - Created, but if that's the case we should add set @ResponseStatus to 201 instead of it's default 200 value.